src/main/java/ListItem.java

```diff
@@ -0,0 +1,62 @@
+public class ListItem {
+
+    private String description;
+    private String status;
+    private boolean isDone = false;
+    private Date date;
+
+
+    ListItem(String description, String command, String date) {
+        try {
+            switch (command) {
+                case "todo":
```

Comment:
Try to conform to the coding style! ( "case ..." should be on the same indent level as the switch statement )